### ISSUE TITLE
"run command" doesn't work anymore

### ISSUE THREAD

**Issue by arnaud-carre** [slynch8/10x#1593]
Hi!
Within a 10x workspace, I used two different batchs for "build command" and "run command". But since one or two version, both "build command" and the "run command" are executing the "build command"! So "run command" is never executed anymore when doing CTRL-F5 

This is a regression as it worked like a charm few days ago

**Comment by slynch8** [comment]
Hi,
I've just tested it and it seems fine. The only thing I changed a few builds back is that it doesn't prepend the working directory to the run command anymore. Could it be that the run command is just failing? (and the build is happening because you have build before run enabled?)

**Comment by slynch8** [comment]
I've just found that it's not using the workspace working directory. I'll get that fixed

**Comment by slynch8** [comment]
I changed the run command to output to the Output panel instead of the Build Output panel, see if there's anything there

**Comment by arnaud-carre** [comment]
"I think it's probably building because you have the BuildBeforeRun setting set?"
probably, but then run.cmd is not executed anymore :( ( it was executed before )

**Comment by slynch8** [comment]
Is it showing anything in the output panel?

**Comment by arnaud-carre** [comment]
nothing. I just tested to set "build before run" to false. And now when doing CTRL-F5, just nothing happen. ( build window doesn't show anything)

**Comment by arnaud-carre** [comment]
also it's not specific to one workspace. I have several 10x workspace with same kind of config (ie m.cmd for build and run.cmd for run). And all these workspace doesn't run anymore with CTRL-F5

**Comment by arnaud-carre** [comment]
oh now I get it! Seems there are two different issues (or change) at the same time:
As you said the run doesn't show anything in build window anymore (that is anoying). It goes to the 10x output window ( which I never use). So now if I show 10x window I get that when CTRL-F5:

```
Executing command:
run.cmd
CreateProcess error: The system cannot find the file specified.
Command result: -1
```

And I guess this is because the working directory regression

I hope you'll put back the "run" output to output window soon :)

**Comment by slynch8** [comment]
ok, I'll move the output back to the build output.

Can you try this build:
https://e.pcloud.link/publink/show?code=XZ9bfSZOa3JKtavbqh5Ef0JwRP5wzFvsj0V

**Comment by slynch8** [comment]
I've added a Run working Directory setting

**Comment by arnaud-carre** [comment]
it works with the build you provided. I mean, run.cmd is properly executed and from the right directory ( ie the same as workspace file ). I didn't configured any new setting
But the output of the run command is still in 10x window ( instead of build window as it was before)

**Comment by slynch8** [comment]
ok, thanks for checking. I'll get that and the output change in the next build

**Comment by arnaud-carre** [comment]
awesome! thanks!

**Comment by slynch8** [comment]
fixed (1.0.7)
